openapi_specgen/openapi.py

```diff
@@ -15,15 +16,18 @@ class OpenApi():
     version = '3.0.2'
     title = None
     paths = None
+    components = None
 
     def __init__(self,
                  title: str,
-                 paths: List[OpenApiPath]):
+                 paths: List[OpenApiPath],
+                 components: ComponentSet):
```

Comment:
Rename this parameter to `authentication` or `security` and make it optional, default value None